pkg/controllers/job/helpers/helpers.go

```diff
@@ -70,3 +71,24 @@ func GenPVCName(jobName string) string {
 func GetJobKeyByReq(req *apis.Request) string {
 	return fmt.Sprintf("%s/%s", req.Namespace, req.JobName)
 }
+
+func GetTaskIndex(taskName string, job *batch.Job) int {
```

Comment:
Pls add comments for functions as format. Same to the whole PR.

Reply:
And l think this func can be named more clearly, maybe `getTasklndexUnderJob `?